src/lib/utils/network.ts

```diff
@@ -0,0 +1,10 @@
+import * as net from "net";
+
+export function findNextPort(port: number) {
```

Comment:
Could you either use this function for the webserver or use the same solution as the webserver?

Basically, I added similar code to the webserver where it tries the port and if it doesn't work, it assigns a free port
https://github.com/embark-framework/embark/blob/cd91cd1f5181ab82a7917272e7a4da7d77059784/src/lib/modules/webserver/index.js#L82

Reply:
I will remove duplication :+1: